lib/src/toggle-group/ToggleGroup.jsx

```diff
@@ -143,33 +199,49 @@ const ToggleContainer = styled.div`
           color: #ffffff;
         }        
         :focus {
-          outline: ${props.theme.focusColor} solid 2px;
-          outline-offset: -2px;
+          border-color: ${props.theme.focusColor};
```

Comment:
Why are we using the `border` property for the focus instead of `outline` or `box-shadow` as the other components? The border property occupies a space while outline and box shadow do not, should that be something to consider here? What do you think, @aweell ?

Reply:
Agree, change the `border-color` to transparent, and add a 2px `box-shadow` to the toggle container (same as we're doing in the `input-text`)

Example:

![focus-within styles for new input-text](https://user-images.githubusercontent.com/44420072/145786739-dda953b8-747d-49ec-8bec-f7d590e397ff.png)

_`focus-within` styles for new `input-text` component_

Reply:
Now `box-shadow` is used instead of `border` .